UPGRADE-1.12.md

```diff
@@ -2,6 +2,13 @@
 
 1. The `Sylius\Component\User\Model\UserInterface` extends the `Symfony\Component\PasswordHasher\Hasher\PasswordHasherAwareInterface`
    interface to fix the compatibility with Symfony 6.
+2. The constructor of `Sylius\Component\Product\ResolverDefaultProductVariantResolver` has been modified, a new argument has been added :
```

Comment:
```suggestion
2. The constructor of `Sylius\Component\Product\DefaultProductVariantResolver.php` has been modified, a new argument has been added :
```

Reply:
```suggestion
2. The constructor of `Sylius\Component\Product\Resolver\DefaultProductVariantResolver` has been modified, a new argument has been added :
```

Looking for other UPGRADE entries, `.php` part is usually not used.

Reply:
I miss the backslash between `Resolver` and `DefaultProductVariantResolver ` ;)